text/0507-subnets.md

```diff
@@ -0,0 +1,437 @@
+# Full control over VPC and subnet configuration
+
+* **Original Author(s):**: @otaviomacedo
+* **Tracking Issue**: #507
+* **API Bar Raiser**: @{BAR_RAISER_USER}
+
+New L2 constructs to allow for greater control over VPC design. Among other
```

Comment:
I think it would be interesting to see a breakdown of all the L1s that are used in creating a Vpc  and which ones we do not have L2 coverage for. To me this breaks down into 3 steps

1. Show how to create the current default `Vpc` with a new complete set of L2s.
2. Show how to create a complicated non-default Vpc with the new set of L2s.
3. Can we make the above two easier with opinionated abstractions

Reply:
I think it might be quite a short list.

- the vpc itself. ( its kind of unforuantate that ec2.Vpc was called Vpc, becuase a vpc in cdk and a vpc in the rest of amazon are two differnet thigns )
- subnets
- route tables.